vue-app/src/store/index.ts

```diff
@@ -278,13 +280,23 @@ const actions = {
   },
   async [LOAD_USER_INFO]({ commit, state }) {
     if (state.currentRound && state.currentUser) {
-      let isVerified = state.currentUser.isVerified
-      if (!isVerified) {
-        isVerified = await isVerifiedUser(
-          state.currentRound.userRegistryAddress,
-          state.currentUser.walletAddress
-        )
+      // Check if this user is in our user registry
+      const isRegistered = await isVerifiedUser(
+        state.currentRound.userRegistryAddress,
+        state.currentUser.walletAddress
+      )
+
+      // If the user is registered, we assume all brightId steps as done
+      let brightId: BrightId = {
+        isLinked: true,
+        isSponsored: true,
+        isVerified: true,
+      }
+      if (!isRegistered) {
+        // Fetch brightId info
+        brightId = await getBrightId(state.currentUser.walletAddress)
```

Comment:
@samajammin do you think this is going to bring us some troubles with the Simple Registry? perhaps I could add a condition to check if we are in the bright Id registry.
```js
if(!isRegistered && isBrightIdRegistry) { ... }
```
But if we leave it as is, I think we are not going to have problems because in case the user is not registered, the brightId info is not going to have any impact on the code.

Reply:
Yes I think we should add a check on `userRegistryType` & only fetch if we're using BrightID: https://github.com/ethereum/clrfund/blob/develop/vue-app/src/api/core.ts#L19

No need to wait for this request if we're using simple registry.

Reply:
Cool! I was looking for that. Yes, makes totally sense.